pkg/services/k8s/apiserver/admission/schema_validate.go

```diff
@@ -3,79 +3,61 @@ package admission
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"io"
 
-	"github.com/grafana/thema"
+	"cuelang.org/go/cue"
+	"github.com/grafana/grafana/pkg/registry/corekind"
 	"github.com/grafana/thema/vmux"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apiserver/pkg/admission"
 
 	"github.com/grafana/grafana/pkg/cuectx"
 	"github.com/grafana/grafana/pkg/infra/log"
-	"github.com/grafana/grafana/pkg/kinds/dashboard"
 )
 
 const PluginNameSchemaValidate = "SchemaValidate"
 
 // Register registers a plugin
-func RegisterSchemaValidate(plugins *admission.Plugins) {
+func RegisterSchemaValidate(plugins *admission.Plugins, reg *corekind.Base) {
 	plugins.Register(PluginNameSchemaValidate, func(config io.Reader) (admission.Interface, error) {
-		return NewSchemaValidate(), nil
+		return NewSchemaValidate(reg), nil
 	})
 }
 
 type schemaValidate struct {
 	log log.Logger
+	reg *corekind.Base
 }
 
 var _ admission.ValidationInterface = schemaValidate{}
 
 // Validate makes an admission decision based on the request attributes.  It is NOT allowed to mutate.
-func (sv schemaValidate) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) (err error) {
+func (st schemaValidate) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) (err error) {
```

Comment:
What was the point of this change?

Reply:
whoops, it was a sloppy/paste that i didn't clean up :)